persistence/src/test/scala/org/apache/pekko/persistence/EventStoreInteropTest.scala

```diff
@@ -346,7 +346,9 @@ object DelayedPersistence {
 class DelayedPersistence extends AsyncWriteJournal {
 
   import DelayedPersistence.*
-  import scala.concurrent.ExecutionContext.Implicits.{global => ec}
+  import scala.concurrent.ExecutionContext.Implicits.global as ec
+
+  Future(IO.delay(()).unsafeRunSync())(ec) // reference `ec` in both Scala 2.13 and 3
```

Comment:
ugly hack to make use of `ec`, the plain import of `global` doesn't work for Scala 2.13, but Scala 3 doesn't see its implicit usage in `Future {}` calls